charts/tool-querybuilder/values.yaml

```diff
@@ -0,0 +1,44 @@
+# Default values for tool-querybuilder.
+
+replicaCount: 1
+
+image:
+  repository: docker-registry.wikimedia.org/repos/wmde/wikidata-query-builder
+  tag: "2024-12-03-110242"
+  pullPolicy: IfNotPresent
+
+platform:
+  mediawikiBackendHost: someHost
+
+app:
+  subclassPropertyMap: '{"default":"P279","P171":"P171","P131":"P131"}'
+  siConversionProperty: 'P2370'
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+resources: {}
```

Comment:
could we specify resources needed for QueryBuilder?

Reply:
We could, per default it's left blank intentionally though. See the auto-generated comment from helm underneath - it's common practice to not define the resources here in the chart (like we do in our other charts), in order to separate this from the configuration where you actually deploy it (thats why we define it in our values files in the deploy repository)